File: public/app/model/inventory/Distributor.js
Ext.define('coloMS.model.inventory.Distributor', {
    extend: 'coloMS.model.inventory.Base',
    
    fields: [
        {
            name: 'address',
            type: 'string'
        },
        {
            name: 'phone',
            type: 'string'
        },
        {
            name: 'email',
            type: 'string'
        }
    ]
});

File: app/models/distributor.rb
class Distributor < ActiveRecord::Base
  attr_accessible :address, :description, :email, :name, :phone
  has_many :orders
  
  validates :name, presence: true, uniqueness: true, length: { maximum: 255 }
  validates :description, length: { maximum: 255 }
  audited
end
